components/brave_rewards/browser/rewards_service_factory.cc

```diff
@@ -35,6 +36,7 @@ RewardsServiceFactory::RewardsServiceFactory()
     : BrowserContextKeyedServiceFactory(
           "RewardsService",
           BrowserContextDependencyManager::GetInstance()) {
+  DependsOn(BitmapFetcherServiceFactory::GetInstance());
```

Comment:
if we end up keep this, both the header and `DependsOn` should be in a `REWARDS_SERVICE_ENABLED` block because the dep is only adding when it is enabled.